src/main/java/com/artipie/Pie.java

```diff
@@ -101,54 +91,7 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
                     storage -> storage.value(new Key.From(String.format("%s.yaml", repo)))
                 )
                 .thenApply(content -> new RepoConfig(this.vertx, content))
-                .thenCompose(Pie::sliceForConfig)
+                .thenCompose(cfg -> CompletableFuture.completedStage(new SliceFromConfig(cfg)))
```

Comment:
@HDouss maybe it would be better to use `.thenApply(cng -> new SliceFromConfig(cfg))` here?

Reply:
@g4s8 Fixed!

Reply:
@g4s8 I don't see the second minor comment..